pip/qsharp/_native.pyi

```diff
@@ -139,12 +139,19 @@ class Interpreter:
         """
         ...
 
-    def run(self, entry_expr: str, output_fn: Callable[[Output], None]) -> Any:
+    def run(
+        self,
+        entry_expr: str,
+        output_fn: Callable[[Output], None],
+        pauli_noise: Optional[Tuple[float, float, float]],
```

Comment:
nitpick: this parameter is called `noise` both above this in _qsharp.py and below this in the Rust code, so why not just call it `noise` here too?

Reply:
I updated.